setup.py

```diff
@@ -46,6 +46,12 @@
         "sphinx-gallery",
         "imageio",
         "matplotlib",
+        # duplicate example dependencies to avoid pyside6
```

Comment:
Maybe PySide6 should not be in the example deps. A few examples are qt-specific, but it can also be e.g. pyqt5. @Korijn ?

Reply:
Can you explain why we want avoid pyside6?

Reply:
@Korijn because we don't need it when building the docs (keeps build times low), and because it started to break the build on RTD.

Reply:
OK.